addons/web/static/src/views/calendar/calendar_controller.scss

```diff
@@ -187,11 +187,19 @@ $o-cw-filter-avatar-size: 20px;
                     height: $o-cw-filter-avatar-size;
                     border-radius: 2px;
                     object-fit: cover;
+                    
+                    .fa-check {
+                        padding-top: 2px;
+                    }
```

Comment:
those changes should not be in planning since it is only in planning we display an icon instead of an image. 🤔

Reply:
Nope in this case it's for the fa-check (checkmark that confirms that the filter is used) in all the current calendar view the checkmark is a little bit too high so i added the padding
![image](https://github.com/odoo/odoo/assets/36259462/fa52ec78-54af-454e-8a29-3f6027377400)

Reply:
Okay thanks for the explanation, let's see with framework js then. 🙂

Reply:
Could you use ´align-items-center´ on ´.o_cw_filter_input_bg.o_beside_avatar´ OR Remove the width and height.